src/signal.cr

```diff
@@ -132,6 +137,28 @@ enum Signal : Int32
     Signal::PIPE.ignore
     Signal::CHLD.reset
   end
+
+  # :nodoc:
+  def self.setup_sigfault_handler
+    return unless @@setup_sigfault_handler.test_and_set
+
+    altstack = LibC::StackT.new
+    action = LibC::Sigaction.new
```

Comment:
Local variables don't need to be declared at the top in Crystal, so I'd move this line down right before `action` is used for the first time for more concise context.